adoc/admin-security-ldap.adoc

```diff
@@ -29,7 +28,6 @@ automatically self-signed certificate and key.
 docker run -d \
 	-p 389:389 \
 	-p 636:636 \
-	-e INSTANCE_NAME=<INSTANCE_NAME> \
 	-e ROOT_DN=<ROOT_DN> \
 	-e ROOT_PASSWORD=<ROOT_PASSWORD> \
 	-e SUFFIX=<SUFFIX> \
```

Comment:
When I look at the new entrypoint.sh script at https://raw.githubusercontent.com/SUSE/caasp-container-images/master/caasp-389-ds-image/root/usr/local/bin/entrypoint.sh I see other variable names.

DS_SUFFIX, DOMAIN_SUFFIX and DS_DM_PASSWORD -> so I assume the ROOT_DN, ROOT_PASSWORD and SUFFIX are also not valid anymore?